docs/getting-started/developers/index.md

```diff
@@ -12,32 +12,46 @@ If you are migrating from Fabric to Warp, please visit the [Migration page](/mig
 
 This is the initial version of Warp and is currently a work in progress. Any elements, components or inline styled/custom CSS that are not implemented using Fabric, is required to be managed in order to make them themeable.
 
+## Before getting started
+When migrating to Warp, this may affect or break the styling for other teams that exists on the same page as your project. **Make sure to communicate and align with affected teams before starting the migration process.**
```

Comment:
Should this be part of the Migration to Warp page? We are linking those who are migrating to Warp from Fabric in the previous header and they might miss this important message. Wdyt?